src/main/java/com/haem/blogbackend/service/CategoryService.java

```diff
@@ -0,0 +1,38 @@
+package com.haem.blogbackend.service;
+
+import com.haem.blogbackend.domain.Category;
+import com.haem.blogbackend.dto.request.CategoryCreateRequestDto;
+import com.haem.blogbackend.dto.response.CategoryResponseDto;
+import com.haem.blogbackend.repository.CategoryRepository;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+@Service
+public class CategoryService {
+    private final CategoryRepository categoryRepository;
+
+    public CategoryService(CategoryRepository categoryRepository) {
+        this.categoryRepository = categoryRepository;
+    }
+
+    @Transactional(readOnly = true)
+    public long getCategoryCount(){
+        return categoryRepository.count();
+    }
```

Comment:
class에 `@Transactional(readOnly = true)` 한번 선언하면 공통적으로 선언 됨
```suggestion
@Transactional(readOnly = true)
@Service
public class CategoryService {
    private final CategoryRepository categoryRepository;

    public CategoryService(CategoryRepository categoryRepository) {
        this.categoryRepository = categoryRepository;
    }

    public long getCategoryCount(){
        return categoryRepository.count();
    }
```